uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/CRM/Tasksassignments/BAO/TaskTest.php

```diff
@@ -69,7 +69,15 @@ function testCreateTaskWithNoTarget() {
 
   function testGetIncompletedStatuses() {
     $incompleteStatuses = Task::getIncompleteStatuses();
+    $allStatuses = array_flip(civicrm_api3('Activity', 'getoptions',
+      ['field' => "activity_status_id"])['values']);
+    $expectedStatuses = [
+      $allStatuses['Scheduled'],
+      $allStatuses['Left Message'],
+      $allStatuses['Unreachable'],
+      $allStatuses['Available']
+    ];
 
-    $this->assertEquals($incompleteStatuses, [1, 4, 5, 7]);
+    $this->assertEquals($incompleteStatuses, $expectedStatuses);
```

Comment:
Not sure if this test will pass on all installations, for example, I have the `Partially Approved` status on my local test db with the grouping column as NULL which is what indicates the incomplete statuses.

Reply:
Done, but as discussed the test looks almost identical to the implementation.